app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -255,7 +257,7 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
         }
       }
     }
-
+    checkRuns = checkRuns ?? <Map<String, dynamic>>[];
```

Comment:
shall this be moved before calling function `_checkStatuses`?

Reply:
Done